unkeyed/unkey
# feat: Eng-334 improve create key UI

## What does this PR do?
eng-334
Changed create key page to layout and style

Fixes #707 

*If there is not an issue for this, please create one first. This is used to tracking purposes and also helps use understand why this PR exists*



## Type of change




## How should this be tested?

Play around with Create Key and make sure it still works as intended. Make sure it only creates valid keys based on options selected

## Checklist



### Required


### Appreciated
